chatty/socket.py

```diff
@@ -1,77 +1,82 @@
 from .evented import Evented
-from ws4py.client.tornadoclient import TornadoWebSocketClient
-from tornado import ioloop
-import random
-import json
 
-# A ws4py websocket wrapped in an Evented class.
-class EventedWebSocket(TornadoWebSocketClient, Evented):
-    def __init__(self, *args, **kwargs):
-        Evented.__init__(self)
-        TornadoWebSocketClient.__init__(self, *args, **kwargs)
+from tornado.websocket import websocket_connect
+from tornado.ioloop import IOLoop
```

Comment:
With 3.x it seems like asyncio is the direction Python is moving for asynchronous capabilities, removing the need for 3rd party event frameworks like Tornado. But this is a bigger change, doesn't have to be in in this PR.